### PR TITLE
Remove vulgar and possibly offensive words

### PR DESCRIPTION
**Pull request by ageis**
This is a significant cleanup of the noun and adjective lists as discussed. The 0.2.1 to 0.3 migration script has its own wordlist which is unaffected.

### CODE REVIEW

**Comment by garrettr** [comment]
> The 0.2.1 to 0.3 migration script has its own wordlist which is unaffected.

:+1: That needs to remain unchanged for the 0.2.1 codenames to migrate correctly.

File: securedrop/dictionaries/nouns.txt [hunk 1]
```diff
@@ -15161,15 +14903,12 @@ slam
...
 slapstick
 slash
 slasher
 slat
 slate
-slaughter
-slave
 slaveholder
 slaver
 slavery
```
Comment (garrettr) [inline securedrop/dictionaries/nouns.txt]: May let's get rid of all `slave*` words?

File: securedrop/dictionaries/nouns.txt [hunk 2]
```diff
@@ -15484,7 +15214,6 @@ span
 spandex
 spandrel
 spaniel
-spank
 spanker
```
Comment (garrettr) [inline securedrop/dictionaries/nouns.txt]: Might as well get rid of "spanker" too

File: securedrop/dictionaries/nouns.txt [hunk 3]
```diff
@@ -15548,15 +15277,12 @@ spelling
 spender
 spending
 spendthrift
-sperm
-spermicide
 sphagnum
 sphere
 sphericity
 spheroid
 sphincter
```
Comment (garrettr) [inline securedrop/dictionaries/nouns.txt]: Let's drop this one, lest we ["Wayne's World"](http://youtu.be/--BslU44vWU) some unsuspecting journalists...

**Comment by ageis** [comment]
Got those and some more. Done for now, merge whenever.

**Comment by garrettr** [comment]
Merging this into release/0.3.1 and develop.

Rationale for merging into 0.3.1: it resolves complaints from live installations that are using the 0.3 code, does not introduce breakage, and does not introduce any security risk.

**Comment by garrettr** [comment]
This was merged in 0.3.1 in bd5e7cf0c5753af425fb92cdf12a7ebaa2da5007 and in develop in 366c657b5f79b3dae510404ea5d6fed6adb0f6ca.

**Pull request closed without merging**